DEV_SETUP.md

```diff
@@ -126,9 +126,9 @@ NOTE: Developers on Mac OS have additional prerequisites. See the [Supplementary
 
   - **Linux**:
 
-    install `default-jre` via apt:
+    install `openjdk-17-jre` via apt:
```

Comment:
Maybe a better change here is to update the heading of this section from `Java (JDK 8)` -> `Java (JDK 17)` ?

Reply:
"may be"? that should definitely be changed, you are too polite @shubham1g5 
thanks for the catch. I will update that and I assume the change in title is along with the change I already made.